AdrKacz/super-duper-guacamole
# Add script to verify Deepsource config

# Add script to verify Deepsource config

Add scripts to enhance DeepSource

1. Ensure all packages are read
2. Verify Test Coverage
